python/pipeline/utils/galvo_corrections.py

```diff
@@ -59,20 +97,20 @@ def plot_raster(filename, key):
     from pipeline import preprocess, experiment
```

Comment:
Can you move the matplotlib and seaborn import into the plot function? I'd prefer to import it only when we plot and not by default.